velox/type/Timestamp.h

```diff
@@ -117,60 +117,20 @@ struct Timestamp {
     return nanos_;
   }
 
-  int64_t toNanos() const {
-    // int64 can store around 292 years in nanos ~ till 2262-04-12.
-    // When an integer overflow occurs in the calculation,
-    // an exception will be thrown.
-    try {
-      return checkedPlus(
-          checkedMultiply(seconds_, (int64_t)1'000'000'000), (int64_t)nanos_);
-    } catch (const std::exception& e) {
-      VELOX_USER_FAIL(
-          "Could not convert Timestamp({}, {}) to nanoseconds, {}",
-          seconds_,
-          nanos_,
-          e.what());
-    }
-  }
+  int64_t toNanos() const;
```

Comment:
We need to keep the definition inlined because the critical path code is fast that function call overhead is not negligible

Reply:
@Yuhta, thanks for your comment! Should I just keep this part unchanged, i.e., keep them in Timestamp.h?

Reply:
Yes keep them in the header to keep them inlined